examples/browser-to-browser/README.md

```diff
@@ -0,0 +1,63 @@
+# js-libp2p-webrtc Browser to Browser
```

Comment:
```suggestion
# js-libp2p-webrtc Browser to Browser
```
I wonder if we should rename this to private-to-private or private browser-to-private browser to be better in line with current rename. Let's discuss in standup tomorrow

Reply:
I could see both sides of this. If we're looking for this example's name to have parity with the spec, then renaming makes sense.  On the other hand, the example is a browser connecting to a browser, so the naming is aligned with the example.  I don't have a strong opinion either way.